codeworld-api/stack.yaml

```diff
@@ -0,0 +1,18 @@
+resolver: lts-14.9
+
+packages:
+  - .
+
+allow-newer: false
+
+extra-deps:
```

Comment:
It's unfortunate that we need to list the dependencies twice here.  Is this really necessary?  Stack cannot get this information from the Cabal file instead?  I thought it still used cabal files for dependencies.

Reply:
No, not necessary.  I was using Stack incorrectly.  Only one `stack.yaml` is necessary.